manual/en/docs/flow/actions/add_hubspot_company_action.md

```diff
@@ -0,0 +1,40 @@
+# Add company to HubSpot plugin
+
+This plugin adds new contact to HubSpot, based on provided data.
+
+## Input
+This plugin takes any payload as input.
+
+## Outputs
+This plugin returns response from HubSpot API on port **response**, or optional
+error info on port **error** if one occurs.
+
+## Plugin configuration
+
+#### With form
+- HubSpot resource - please select your HubSpot resource. It should contain: 
+    - HubSpot client id
```

Comment:
Please add where in HubSpot do I find the information on client-id and client-secret. If there is info on this in HubSpot documentation you could paste it here.

Reply:
Hi
I've realised, that I was too inspired by Mautic plugins, I think I've fixed the mistake, because in HubSpot you need to initiate an OAuth connection (IMO it is better to do that without Tracardi, because you need to go to app auth anyway). I've added a situation where you heven't got an access token yet and there are small changes in the rest of the code
I think the docs in the .md file are detailed and understandable
I have an idea - how about making the .md file with initiating OAuth connection an separated file?
I think it could make the docs more transparent